src/mixins/animation-frame.html

```diff
@@ -0,0 +1,64 @@
+<script>
+  window.requestAnimFrame = (function(){
```

Comment:
We can use standard - `window.requestAnimationFrame`
http://caniuse.com/#search=requestAnimationFrame